test/fbe/test_octo.rb

```diff
@@ -162,4 +162,39 @@ def test_pauses_when_quota_is_exceeded
     end
     assert_in_delta(pause, Time.now - start_time, 1)
   end
+
+  def test_fetches_fake_check_runs_for_ref
+    WebMock.disable_net_connect!
+    o = Fbe.octo(loog: Loog::NULL, global: {}, options: Judges::Options.new({ 'testing' => true }))
+    result = o.check_runs_for_ref('zerocracy/baza', 'sha')
```

Comment:
@Suban05 Maybe instead `'sha'` string use something more reminiscent of the hash string? 
E. g.
```ruby
result = o.check_runs_for_ref('zerocracy/baza', 'f2ca1bb6c7e907d06')
```